useKeyPress/test.jsx

```diff
@@ -0,0 +1,49 @@
+//And here is a test case to validate the hook:
+
+import React from "react";
+import { render, fireEvent, cleanup } from "@testing-library/react";
+import useKeyPress from "./useKeyPress";
+
+describe("useKeyPress", () => {
+  afterEach(cleanup);
+
+  it("returns true when the target key is being pressed", () => {
+    const TestComponent = () => {
+      const enterPressed = useKeyPress("Enter");
+
+      return <div>{enterPressed.toString()}</div>;
+    };
+
+    const { getByText } = render(<TestComponent />);
```

Comment:
Since we're doing this same kind of thing in each test, why don't we create a `setup` function that looks something like this:
```ts
function setup () {
  const TestComponent = (): FC => {
    const enterPressed = useKeyPress("Enter")

    return <div>{enterPressed.toString()}</div>
  }

  const utils = render(<TestComponent />)

  return { ...utils }
}
```
And then in our tests:
```ts
it('returns true when the target key is being pressed', () => {
  const { getByText } = setup()

  // do your stuff

  // make your assertion
})
```